src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -124,7 +134,7 @@ def _f1_score(
     """
     model_output = _normalize_and_strip_text(model_output, normalize_text=normalize_text, strip_text=strip_text)
     target_output = _normalize_and_strip_text(target_output, normalize_text=normalize_text, strip_text=strip_text)
-    ret = f_measure(reference=set(target_output.split(" ")), test=set(model_output.split(" ")))
+    ret = f_measure(reference=set(_split(target_output)), test=set(_split(model_output)))
```

Comment:
Do we really need a separate function? Could we not replace `split(" ")` with `split()`. Simply using `split()` gives the desired behavior that you specified in the unit tests.

Reply:
Ah makes sense. I didn't know the behavior of split with no separator. Actually from the docs the behavior is not that clear. https://docs.python.org/3.10/library/stdtypes.html?highlight=str%20split#str.split  
there is no mention about newlines and other special characters. But yes str.split() seems to achieve precisely what we need. Will change.

Reply:
The only thing `_split` does is to call the standard library `split()` function, so consider calling `split()` directly. Its just one more level that the reader has to go through and not sure that it adds any clarity since the user can simply look at the documentation of standard library `split()` to understand what it does.